.github/workflows/bump-elastic-stack-version.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: bump-elastic-stack-version
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 15 * * 1-5'
+
+permissions:
+  contents: read
+
+jobs:
+  bump-elastic-stack:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    needs: [filter]
```

Comment:
There is no jobs `filter` defined. Probably, this job fails or could not start. Is it required another job here ? 

I guess it is probably based on this workflow: https://github.com/elastic/beats/blob/d23b4d33622137f7245b628d77d15f8ecfef3a0d/.github/workflows/bump-elastic-stack-snapshot.yml#L16

Currently, the `filter` job looks like would not be needed. If so, this could be removed.

Reply:
Yep, this is a leftover from copying from beats.